### DISCUSSION TITLE
MSI Installer

### DISCUSSION THREAD

**Discussion by bmcgeorge-winconn** [adoptium/adoptium-support#1140] [Q&A] [answered]
Hi,

When installed from the command line, the MSI installer doesn't seem to be adding itself to the Add/Remove programs list (Programs and Features) on Windows.  I've installed on multiple systems and our RMM tool continues to not pickup the installation.  It is there though (I can see it in Windows Explorer).

This is not a huge issue, but it is an annoyance.

**Comment by karianna** [comment]
@bmcgeorge-winconn Which is the exact installer that you're using and what command are you running?

**Reply by bmcgeorge-winconn** [reply]
I was using the OpenJDK21U-jdk_x64_windows_hotspot_21.0.1_12.msi with this command:

msiexec /i \\srv-dc1\shared\software\OpenJDK21U-jdk_x64_windows_hotspot_21.0.1_12.msi ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome,FeatureOracleJavaSoft INSTALLDIR="C:\Program Files\Eclipse Adoptium\" /quiet

I switched to the jre installer OpenJDK21U-jre_x64_windows_hotspot_21.0.3_9.msi with this command:

msiexec /i \\srv-dc1\shared\software\OpenJDK21U-jre_x64_windows_hotspot_21.0.3_9.msi ADDLOCAL=FeatureMain,FeatureEnvironment,FeatureJarFileRunWith,FeatureJavaHome,FeatureOracleJavaSoft INSTALLDIR="C:\Program Files\Eclipse Adoptium\jre-21.0.3_9-hotspot\" /quiet

The JRE seems to be showing up in the RMM tool.

**Reply by bmcgeorge-winconn** [reply, marked as answer]
Nevermind.  It looks like it's showing up now.  Not sure why it took so long, but it's working.

Thanks for your time!
